consul-enterpise-admin-partitions-eks/variables.tf

```diff
@@ -0,0 +1,63 @@
+variable "eks_cluster_name_primary" {
+  description = "Name of the primary EKS Cluster that runs the Consul Enterprise server cluster"
+  type        = string
+  default     = "primary"
+}
+variable "eks_cluster_name_secondary" {
+  type        = string
+  description = "Name of the secondary EKS Cluster that runs the Consul Enterprise client cluster"
+  default     = "secondary"
+}
+variable "license_name" {
+  type        = string
+  description = "The name of the Consul Enterprise license file. Place this file in the `./consul_enterprise/` directory"
+  default     = "consul.hclic"
+}
+
+# Default tags to pass to AWS. You can set them here, or in a tfvars file.
+variable "default_tags" {
+  type = map(string)
+}
```

Comment:
This will fail unless the user provides a set of values. We should at the very least put some defaults here so that the enduser can run this without making changes.